lib/cli_common/requirements-dev.txt

```diff
@@ -1,3 +1,31 @@
+# linting
 flake8
+flake8-coding
+flake8-copyright
+flake8-debugger
 flake8-isort
+flake8-mypy
+flake8-quotes
+#flake8-todo  # TODO: we will need to first remove all TODOs from the code
```

Comment:
I don't mind TODOs in the code as long as they are linked to an issue (otherwise you risk forgetting about them), but I don't have a strong opinion here.

Reply:
I also thought this to be a little too controversial that is also the reason why I commented it out and ironically created a `# TODO: ` comment :smile: 

I think I agree with @marco-c here that we should always create an issue and not use source code as an issue tracker. Some of these TODOs are hard to find, most likely we forget about them and we could actually tag some of them as `good-first-bugs`.

I do find using TODOs in source code useful when creating something bigger, but this should always be cleanup before landing on master. We could expand [flake8-todo](https://github.com/schlamar/flake8-todo/blob/master/flake8_todo.py) to allow TODOs if they contain link to the issue tracker.

I think this sounds like an interesting `good-first-bug` for somebody to learn slowly about services. What do you think?